lib/utils/constants.dart

```diff
@@ -0,0 +1 @@
+const liquidMinimumPaymentAmountSat = 1000;
```

Comment:
Don't we expose that in the sdk? Sounds like it needs to be there.

Reply:
Yes, I agree. We need to fill that gap in the sdk.

Reply:
Seems like we can get this value from `fetchLightningLimits` sdk function

Reply:
Addressed on child PR
- Fetch lightning limits on initializing LnUrlCubit ac992dc
- Rely on LnUrlState limits when validating payments 3ce88b9